src/runtime-rs/crates/runtimes/linux_container/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+use std::sync::Arc;
+
+use anyhow::Result;
+use async_trait::async_trait;
+use common::{message::Message, ContainerManager, RuntimeHandler, Sandbox};
+use tokio::sync::mpsc::Sender;
+
+unsafe impl Send for LinuxContainer {}
+unsafe impl Sync for LinuxContainer {}
+pub struct LinuxContainer {}
+
+pub fn init() -> Result<()> {
+    Ok(())
+}
+
+pub fn name() -> String {
```

Comment:
why not putting `name()` in `RuntimeHandler` too?

Reply:
Thanks for your review, putting `name()` `init()` to `RuntimeHandler`.